### PR TITLE
remove user view

### PR DESCRIPTION
**Pull request by rosasck**
You can no longer see the user view for both researchers and students

### CODE REVIEW

File: simcon_project/simcon_project/urls.py [hunk 1]
```diff
@@ -28,25 +28,31 @@
...
     path('student/', student_view, name="student-view"),
-    path('student/settings/', student_settings_view, name="student-settings-view"),
-    path('student/register/<uidb64>/', student_registration, name="student-registration"),
-    path('student/conversation/', include('conversation_templates.urls.conv_urls'), name='conversation'),
+    path('student/settings/', student_settings_view,
+         name="student-settings-view"),
+    path('student/register/<uidb64>/',
+         student_registration, name="student-registration"),
+    path('student/conversation/',
+         include('conversation_templates.urls.conv_urls'), name='conversation'),
```
Comment (Mkay971) [inline simcon_project/simcon_project/urls.py]: I know its a little nit picky but would you be able to move these back to the single line like it was before?

Comment (Mkay971) [reply simcon_project/simcon_project/urls.py]: and the rest of them as well.

Comment (rosasck) [reply simcon_project/simcon_project/urls.py]: i honestly cant, somehow prettier wont uninstall, I messed around with my settings and i cant get it to overwrite

Comment (rosasck) [reply simcon_project/simcon_project/urls.py]: i fixed it, i ended up keybinding and shit..

**Review by Mkay971 — APPROVED** [review]
looks good.

**Review by ericharper1 — APPROVED** [review]
Solid!

**Review by kaigoos — APPROVED** [review]

**Pull request merged**